cfgov/jinja2/v1/rural-or-underserved/index.html

```diff
@@ -294,20 +294,24 @@ <h2 class="js-table-title">
                             First-lien covered transactions extended in <span class="chosenYear">2015</span> that are secured by these properties can be counted as rural or underserved in your determination of your first-lien covered transactions in <span class="chosenYear">2015</span> that are secured by properties located in rural or underserved areas.
                         </p>
 
-                        <table class="table" style="border-width: 1px;" width="100%">
```

Comment:
Can we restore the 100% width to all the tables and still keep them scrollable? I think for Wagtail-generated tables we do it with a `class=u-w100pct` on the `table`, but I'm not sure if that will conflict with the scrollable variation. I think making all the tables 100% width again will help with quickly scanning the page since it puts the columns in the same position across all four sections, which means your eye won't have to jump around as much to see the same columns as you scroll.

Reply:
Hmm, 100% is set on `o-table` [here](https://github.com/cfpb/capital-framework/blob/master/packages/cf-tables/src/cf-tables.less#L118), but the scrollable table wraps the table in a div, so `o-table` ends up on the container, not the table. I don't particularly like having two table patterns where `o-table` is applied, so this may need to get threshed out in cf-tables. But in the mean time, we can add it here.

Reply:
I overrode the table width and added a TODO for adjusting the styles in cf-tables.

Reply:
Looks good 👍